docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1714 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #73.            if len(non_trivial_cands_idx) == 0:
</pre><p>Similar to an earlier comment, can we do:</p><p> </p><pre class="ql-syntax" spellcheck="false">if len(non_trivial) &gt; 0:
    cand_idx_chunks = _get_chunks_ranges(non_trivial_cands_idx, shift=m-1)&nbsp;
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Yes... I need to do some transfer learning in my mind 😄 Will be revised.